tests/Unit/Elliptic/Systems/Poisson/Test_Equations.cpp

```diff
@@ -0,0 +1,413 @@
+    tnsr::I<DataVector, Dim, Frame::Inertial>
+        reversed_normal_dot_numerical_flux_for_aux_field(num_points, 0.0);
+    apply_numerical_flux(
+        flux_computer, packaged_data_ext, packaged_data_int,
+        make_not_null(&reversed_normal_dot_numerical_flux_for_field),
+        make_not_null(&reversed_normal_dot_numerical_flux_for_aux_field));
+
+    CHECK_ITERABLE_APPROX(get(normal_dot_numerical_flux_for_field),
+                          -get(reversed_normal_dot_numerical_flux_for_field));
+    for (size_t d = 0; d < Dim; ++d) {
+      CHECK_ITERABLE_APPROX(
+          normal_dot_numerical_flux_for_aux_field.get(d),
+          -reversed_normal_dot_numerical_flux_for_aux_field.get(d));
+    }
+  }
+
+  // Explicit data
+  {
+    flux_computer.package_data(make_not_null(&packaged_data_int),
+                               get<Poisson::Field>(field_variables), grad_field,
+                               unit_normal);
+    flux_computer.package_data(make_not_null(&packaged_data_ext), ext_field,
+                               ext_grad_field, opposite_unit_normal);
+    apply_numerical_flux(
+        flux_computer, packaged_data_int, packaged_data_ext,
+        make_not_null(&normal_dot_numerical_flux_for_field),
+        make_not_null(&normal_dot_numerical_flux_for_aux_field));
+
+    Scalar<DataVector> expected_normal_dot_numerical_flux_for_field(num_points,
+                                                                    0.0);
+    tnsr::I<DataVector, Dim, Frame::Inertial>
+        expected_normal_dot_numerical_flux_for_aux_field(num_points, 0.0);
+    get(expected_normal_dot_numerical_flux_for_field) =
+        -10. * (get(get<Poisson::Field>(field_variables)) - get(ext_field));
+    for (size_t d = 0; d < Dim; d++) {
+      get(expected_normal_dot_numerical_flux_for_field) +=
+          0.5 * unit_normal.get(d) *
+          (grad_field.get(d) + ext_grad_field.get(d));
+      expected_normal_dot_numerical_flux_for_aux_field.get(d) =
+          -0.5 * unit_normal.get(d) *
+          (get(get<Poisson::Field>(field_variables)) + get(ext_field));
+    }
+
+    CHECK_ITERABLE_APPROX(get(normal_dot_numerical_flux_for_field),
+                          get(expected_normal_dot_numerical_flux_for_field));
+    for (size_t d = 0; d < Dim; ++d) {
+      CHECK_ITERABLE_APPROX(
+          normal_dot_numerical_flux_for_aux_field.get(d),
+          expected_normal_dot_numerical_flux_for_aux_field.get(d));
+    }
+  }
+}
+}  // namespace
+
+template <size_t Dim>
+using simple_tags = db::AddSimpleTags<Tags::Mesh<Dim>, Tags::ElementMap<Dim>>;
+template <size_t Dim>
+using compute_tags = db::AddComputeTags<
+    Tags::LogicalCoordinates<Dim>,
+    Tags::MappedCoordinates<Tags::ElementMap<Dim>,
+                            Tags::Coordinates<Dim, Frame::Logical>>,
+    Tags::InverseJacobian<Tags::ElementMap<Dim>,
+                          Tags::Coordinates<Dim, Frame::Logical>>>;
+
+SPECTRE_TEST_CASE("Unit.Elliptic.Systems.Poisson.Mixed.Volume",
```

Comment:
Would the time for the tests be below the timeout if you only have one test case? If so, I think that's better because we spend most of our testing time starting and shutting down the executable currently.